src/models/client_config.ts

```diff
@@ -6,10 +6,12 @@ export function ConvertToClientConfig(raw: string): ClientConfig {
 
 /* tslint:disable:variable-name */
 export class ClientConfig {
+  public api_url: string;
+  public ui_url: string;
+
   public user?: string;
   public api_key?: string;
-  public api_url?: string;
-  public ui_url?: string;
+  public bugsnag_key?: string;
```

Comment:
Do I need to have a Bugsnag key to test this out? If so, how do I get one?

Reply:
I actually switched this to get evergreen's token from the db, rather than having to specify one in the config. It should work now without you knowing it